effects/genericunitexplosion.lua

```diff
@@ -3058,7 +3058,7 @@ definitions['decoycommander'].confetti = {
 	unit               = false,
 	properties = {
 		airdrag            = 0.95,
-		colormap           = [[1 0 0 0.01  0 1 0 0.01 0.1  1 0 0 0.01 0.1  0 1 0.5 0.01  1 0 0.01  0 1 0 0.01  1 1 0 0.01  1 0 0.01   1 0.6 0.01  0 1 0 0.01 0.1   0.1 0.7 0.01   0.6 0.3 0.6 0.01   0.6 0.6 0.6 0.01   0 0 0 0.01  ]],
+		colormap           = [[1 0 0 0.01  0 1 0 0.01 0.1  1 0 0 0.01 0.1  0 1 0.5 0 0.01  1 0 0 0.01  0 1 0 0.01  1 1 0 0.01  1 0 0 0.01   1 0.6 0 0.01  0 1 0 0.01 0.1   0.1 0.7 0 0.01   0.6 0.3 0.6 0.01   0.6 0.6 0.6 0.01   0 0 0 0.01  ]],
```

Comment:
you've got a bunch of groups of 5 and are introducing a new one
![image](https://github.com/user-attachments/assets/7b671dd7-45f2-41ca-b08d-3cd92536ff16)

Reply:
Thank you sprunk! It was the end of the day!
Shit, now my hands are dirty!